composer.json

```diff
@@ -14,7 +14,7 @@
     ],
     "require": {
         "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "^6.3",
+        "guzzlehttp/guzzle": "^6.3|^7.0.1",
```

Comment:
it might should be a double pipe `||` instead of a single `|`

Reply:
Thank you for your comments. I checked the Composer documentation and source code. Single | is indeed a non-standard way of writing, but it can be supported. I have modified